api/utils/grpc/stream/stream.go

```diff
@@ -110,7 +139,7 @@ func (c *ReadWriter) Write(b []byte) (int, error) {
 	var sent int
 	for len(b) > 0 {
 		chunk := b
-		if len(chunk) > MaxChunkSize {
+		if !c.options.DisableChunking && len(chunk) > MaxChunkSize {
```

Comment:
The comment for the `Write` method needs to be updated, now that there's a difference in behavior.